tapir/statistics/views.py

```diff
@@ -206,6 +217,62 @@ def get_context_data(self, **kwargs):
         return context_data
 
 
+class AverageBasketEvolutionJsonView(LoginRequiredMixin, JSONView):
+    def get_context_data(self, **kwargs):
+        tapir_user = get_object_or_404(TapirUser, pk=(self.kwargs["pk"]))
+        logged_user = self.request.user
+        if tapir_user.pk != logged_user.pk and not logged_user.has_perm(
+            PERMISSION_COOP_MANAGE
```

Comment:
I'm actually not sure about this line - maybe it should be another permission?

Reply:
I think PERMISSION_ACCOUNTS_VIEW is more appropriate.
Also, you can use the PermissionRequiredMixin to check for permissions, you can then reuse the same logic as the the TapirUserDetailView : https://github.com/SuperCoopBerlin/tapir/blob/5ef7e14ee2680cba2ae68bcf7c93ffc0f7b46ea9/tapir/accounts/views.py#L49-L52